.github/workflows/pypi-release.yaml

```
@@ -0,0 +1,27 @@
+# This workflow will release project to PyPI
+name: CloudFormation CLI Java Plugin Release
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    if: endsWith(github.ref, "-plugin")
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip wheel twine
```

Not sure if we need to `pip install pip` here:
```suggestion
        pip install --upgrade wheel twine
```